src/votekit/plots/mds.py

```diff
@@ -65,23 +67,41 @@ def plot_MDS(
         dissimilarity="precomputed",
         n_jobs=1,
         normalized_stress="auto",
+        random_state=random_seed,
     )
     pos = mds.fit(np.array(dist_matrix)).embedding_
 
-    # Plot and color data
-    fig, ax = plt.subplots()
-
+    coord_dict = {}
     start_pos = 0
     for key, value_list in data.items():
+        # color, label, marker = key
         end_pos = start_pos + len(value_list)
-        ax.scatter(
-            pos[start_pos:end_pos, 0],
-            pos[start_pos:end_pos, 1],
-            color=key,
-            lw=0,
-            s=marker_size,
-        )
+        coord_dict[key] = (pos[start_pos:end_pos, 0], pos[start_pos:end_pos, 1])
         start_pos += len(value_list)
+
+    return coord_dict
+
+
+def plot_MDS(coord_dict: dict):
```

Comment:
The latter is what I had envisioned, but mostly because that's my personal preference for dealing with matplotlib things. Would be happy to learn how to args or kwargs my way into being able to edit the plot easier.